.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.DS_Store
```

Comment:
This is an OSX hidden file which keeps track of things in the background and is typically in every gitignore.